pkg/cmd/cli/cmd/tag.go

```diff
@@ -389,6 +389,9 @@ func (o TagOptions) RunTag() error {
 			default:
 				istag.Tag.From.Name = localRef.NameString()
 				istag.Tag.From.Namespace = o.ref.Namespace
+				if len(o.ref.Namespace) == 0 {
```

Comment:
@smarterclayton 
> Actually, the from should not have to be set if you are in the current namespace.

Hm, it seems to be exiting with the same error described on this PR [here](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/tag.go#L419) if I do not set a value in `istag.Tag.From.Namespace` though